packages/mask/src/plugins/Tips/components/TipForm.tsx

```diff
@@ -177,7 +177,7 @@ export const TipForm: FC<Props> = memo(({ className, onAddToken, onSent, ...rest
 
             <PluginWalletStatusBar>
                 <ChainBoundary
-                    expectedPluginID={pluginId}
+                    expectedPluginID={pluginId === NetworkPluginID.PLUGIN_FLOW ? NetworkPluginID.PLUGIN_EVM : pluginId}
```

Comment:
A little confuse at the first glance. Recommend to use whitelist mode

```suggestion
                    expectedPluginID={[NetworkPluginID.PLUGIN_EVM, NetworkPluginID.PLUGIN_SOLANA].includes(pluginId) ? pluginId : NetworkPluginID.PLUGIN_EVM}
```